external/solr/src/test/resources/cores/docs/conf/schema.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Requires a license header. You can just run with the `rat` profile locally, so you see all the complains ;-)